yt_dlp/extractor/onenewsnz.py

```diff
@@ -0,0 +1,95 @@
+                'description': 'Greenpeace accused the Government of "greenwashing" instead of taking climate action.',
+
+            },
+        }, {
+            # YouTube video
+            'url': 'https://www.1news.co.nz/2022/09/30/now-is-the-time-to-care-about-womens-rugby/',
+            'info_dict': {
+                'id': 's4wEB9neTfU',
+                'title': 'Why I love women’s rugby: Black Fern Ruahei Demant',
+                'ext': 'mp4',
+                'channel_follower_count': int,
+                'channel_url': 'https://www.youtube.com/channel/UC2BQ3U9IxoYIJyulv0bN5PQ',
+                'tags': 'count:12',
+                'uploader': 'Re: News',
+                'upload_date': '20211215',
+                'uploader_id': 'UC2BQ3U9IxoYIJyulv0bN5PQ',
+                'uploader_url': 'http://www.youtube.com/channel/UC2BQ3U9IxoYIJyulv0bN5PQ',
+                'channel_id': 'UC2BQ3U9IxoYIJyulv0bN5PQ',
+                'channel': 'Re: News',
+                'like_count': int,
+                'thumbnail': 'https://i.ytimg.com/vi/s4wEB9neTfU/maxresdefault.jpg',
+                'age_limit': 0,
+                'view_count': int,
+                'categories': ['Sports'],
+                'duration': 222,
+                'description': 'md5:8874410e5740ed1d8fd0df839f849813',
+                'availability': 'public',
+                'playable_in_embed': True,
+                'live_status': 'not_live',
+            }
+        }, {
+            # 2 Brightcove videos
+            'url': 'https://www.1news.co.nz/2022/09/29/raw-videos-capture-hurricane-ians-fury-as-it-slams-florida/',
+            'info_dict': {
+                'id': 'raw-videos-capture-hurricane-ians-fury-as-it-slams-florida',
+            },
+            'playlist_mincount': 2,
+        }, {
+            'url': 'https://www.onenews.co.nz/2022/09/29/cows-painted-green-on-parliament-lawn-in-climate-protest/',
+            'only_matching': True,
+        }]
+
+    BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/%s/0xpHIR6IB_default/index.html?videoId=%s'
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        fusion_metadata = self._search_json(r'Fusion\.globalContent\s*=\s*', webpage, 'fusion metadata', display_id)
+
+        entries = []
+        for item in traverse_obj(fusion_metadata, 'content_elements') or []:
+            item_type = traverse_obj(item, 'subtype')
+            if item_type == 'video':
+                brightcove_config = traverse_obj(item, ('embed', 'config'))
+                brightcove_url = self.BRIGHTCOVE_URL_TEMPLATE % (
+                    traverse_obj(brightcove_config, 'brightcoveAccount') or '963482464001',
+                    traverse_obj(brightcove_config, 'brightcoveVideoId')
+                )
+                entries.append(self.url_result(brightcove_url, ie='BrightcoveNew'))
```

Comment:
Would prefer if u import the IE and do
```suggestion
                entries.append(self.url_result(brightcove_url, BrightcoveNewIE))
```
This makes it easier to refactor if the ie name ever changes. For youtube/generic, string's fine since we'll probably never change them